k9mail/src/main/java/com/fsck/k9/view/MessageCryptoDisplayStatus.java

```diff
@@ -156,6 +168,9 @@
     @StringRes public final Integer textResTop;
     @StringRes public final Integer textResBottom;
 
+    /** max acceptable time between email timestamp and signature timestamp in millisecs */
+    private static final long MAX_ACCEPTABLE_TIME_DELTA_OF_SIGNATURE = 1000*60*60*24*2; // 2d
```

Comment:
For readability without comment: ``https://developer.android.com/reference/android/text/format/DateUtils.html#DAY_IN_MILLIS * 2``